app/src/main/java/org/mozilla/focus/settings/SettingsFragment.java

```diff
@@ -20,23 +20,5 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         addPreferencesFromResource(R.xml.settings);
-
-        setupPreferences(getPreferenceScreen());
-    }
-
-    private void setupPreferences(PreferenceGroup group) {
-        for (int i = 0; i < group.getPreferenceCount(); i++) {
-            final Preference preference = group.getPreference(i);
-
-            if (preference instanceof PreferenceGroup) {
-                setupPreferences((PreferenceGroup) preference);
-            }
-
-            if (preference instanceof DefaultBrowserPreference
-                    && !((DefaultBrowserPreference) preference).shouldBeVisible()) {
-                group.removePreference(preference);
-                i--;
-            }
-        }
```

Comment:
Do we still need this? AFAICT we now show the DefaultBrowserPreference on all Android versions, so we don't need to manually remove it?